packages/types/src/replay.ts

```diff
@@ -10,6 +10,7 @@ export interface ReplayEvent extends Event {
   trace_ids: string[];
   replay_id: string;
   segment_id: number;
+  replay_type: 'session' | 'error';
```

Comment:
This is of type `ReplayRecordingMode`, but that type is defined in `replays/types` and would probably cause circular import issues -- should we move that type into `types` package?

Reply:
hmm I think it should be fine to use it from `replay/types.ts`, as we do not import from `src/replay` from there?

Reply:
Yes, I think that makes sense! I'll move it there and get this PR merged (so we unblock master) 👍